openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -42,28 +62,40 @@ export class CourseOutline {  // eslint-disable-line import/prefer-default-expor
 
           sections.forEach(section => section.addEventListener('click', (event) => {
             const sectionToggleButton = event.currentTarget;
-            const $toggleButtonChevron = $(sectionToggleButton).children('.fa-chevron-right');
-
             if (sectionToggleButton.classList.contains('accordion-trigger')) {
               const isExpanded = sectionToggleButton.getAttribute('aria-expanded') === 'true';
-              const $contentPanel = $(document.getElementById(sectionToggleButton.getAttribute('aria-controls')));
-
               if (!isExpanded) {
-                $contentPanel.slideDown();
-                $contentPanel.removeClass('is-hidden');
-                $toggleButtonChevron.addClass('fa-rotate-90');
-                sectionToggleButton.setAttribute('aria-expanded', 'true');
+                  expandSection(sectionToggleButton);
               } else if (isExpanded) {
-                $contentPanel.slideUp();
-                $contentPanel.addClass('is-hidden');
-                $toggleButtonChevron.removeClass('fa-rotate-90');
-                sectionToggleButton.setAttribute('aria-expanded', 'false');
+                  collapseSection(sectionToggleButton);
               }
-
               event.stopImmediatePropagation();
             }
           }));
         });
+
+      const toggleAllButton = document.querySelector('#expand-collapse-outline-all-button');
+      const toggleAllSpan = document.querySelector('#expand-collapse-outline-all-span');
```

Comment:
Change this to ```document.querySelector('#expand-collapse-outline-all-span').innerText``` and rename to toggleAllButtonText or something similar?

Reply:
I don't think that'll work (even if this was a `var` and not a `const`).  Doing that assignment would end up giving me `toggleAllSpan = "Expand All"`.  Changing the value of the variable later won't change the value of the element's `innerText` attribute.